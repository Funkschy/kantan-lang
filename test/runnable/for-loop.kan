import "io";

def main(): void {
    let i = 100;
    for let i = 0; i < 3; i += 1 {
        io.printf("%d", i+1);
    }
    io.printf("\n");
    for let i = 3; i > 0; i -= 1 {
        io.printf("%d", i);
    }
    io.printf("\n");
    for ;i < 101; {
        io.printf("%d\n", i);
        i = i + 1;
    }
}
