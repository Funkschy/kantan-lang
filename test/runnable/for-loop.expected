123
321
100
